devolutions-agent/Cargo.toml

```diff
@@ -17,7 +17,8 @@ ctrlc = "3.4"
 devolutions-agent-shared = { path = "../crates/devolutions-agent-shared" }
 devolutions-gateway-task = { path = "../crates/devolutions-gateway-task" }
 devolutions-log = { path = "../crates/devolutions-log" }
-now-proto-pdu = { path = "../crates/now-proto-pdu" }
+# TODO(@pacmancoder): This should point to IronRDP repo after `now-proto-pdu` code is merged to `IronRDP` codebase
+# now-proto-pdu = { path = "../crates/now-proto-pdu" }
```

Comment:
My understanding is that we are not going to use this crate in the agent at all.
```suggestion
```